src/front-end/src/components/MenteeForm.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable */

```

Comment:
We don't want to disable the linter on a whole component. Let's remove this and fix any linting errors that arise.

Reply:
done